README.md

```diff
@@ -22,27 +22,31 @@ https://zapplanner.albylabs.com/confirm?amount=21&recipient=hello@getalby.com&ti
 
 ## Installation
 
-Run `yarn install`
+Run `$ yarn install`
 
-Run `cp .env.example .env.local && husky install`
+Run `$ cp .env.example .env && husky install`
 
-Run `yarn cloak:generate` and set `PRISMA_FIELD_ENCRYPTION_KEY=<CLOAK_MASTER_KEY>` in .env.local
+Run `$ yarn cloak:generate` and set `PRISMA_FIELD_ENCRYPTION_KEY=<CLOAK_MASTER_KEY>` in `.env`
 
-Run `yarn db:migrate:deploy` (if developing with Docker make sure to run Run `yarn docker:start` first)
+Unless you already have it, create the database with: `$ createdb boostagram-viewer`
```

Comment:
These instructions are only needed if you don't use docker right? maybe we should split the installation instructions like the development ones are below

Reply:
I didn't use docker, yes.